x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "apm_jsbase",
+  "title": "APM: RUM Javascript",
+  "description": "Detect anomalies in spans and errors",
```

Comment:
Detect problematic spans and identify user agents that are potentially causing issues.

Reply:
Fixed in https://github.com/elastic/kibana/pull/53792/commits/144c0ba1c62d4da9d1fc35a03d930a1537bfb205